website/project/model.py

```diff
@@ -760,7 +757,7 @@ def __init__(self, reason, key, *args, **kwargs):
 
 def validate_doi(value):
     # DOI must start with 10 and have a slash in it - avoided getting too complicated
-    if not re.match('10\\.\\S*\\/', value):
+    if value and not re.match('10\\.\\S*\\/', value):
```

Comment:
There should have been a ticket on Preprints that added yesterday based on talk in the Preprints flow. Could you see if you think this will take care of it and, if so, link the PR to that ticket for testing?